vcd/resource_vcd_org_group.go

```diff
@@ -0,0 +1,198 @@
+package vcd
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+func resourceVcdOrgGroup() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceVcdOrgGroupCreate,
+		Read:   resourceVcdOrgGroupRead,
+		Update: resourceVcdOrgGroupUpdate,
+		Delete: resourceVcdOrgGroupDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceVcdOrgGroupImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"org": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Description: "The name of organization to use, optional if defined at provider " +
+					"level. Useful when connected as sysadmin working across different organizations",
+			},
+			"name": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true, // vCD does not allow to change group name
+				Description: "SAML group name",
+			},
+			"provider_type": &schema.Schema{
```

Comment:
I thought about suggesting to rename this to `identity_provider`or `identity_provider_type`, so it's instantly visible from HCL about which provider are we talking about and there's less conflict with Terraform provider concept. However, I have realized that `org_user`is already using this field name and consistency between these two occurrences is more important:
https://www.terraform.io/docs/providers/vcd/r/org_user.html#provider_type

CC: @dataclouder @vbauzysvmware